src/components/shared/header.jsx

```diff
@@ -18,10 +19,13 @@ class Header extends Component {
   constructor(props) {
     super(props);
 
-    this.state = { showModal: false };
+    this.state = { showModal: false, showSignUpModal: false };
```

Comment:
Please rename old `showModal` to `showLoginModal`. Make sure the functionality is still running fine.